src/utils/api.test.ts

```diff
@@ -3,16 +3,18 @@ import api, { handleResponseType } from '../../src/utils/api';
 describe('api', () => {
   describe('handleResponseType', () => {
     test('should handle missing Content-Type', async () => {
+      const responseText = `responseText`;
+      const ok = false;
```

Comment:
you could move this line to the object below 😉

Reply:
Yes but I kept them separate because `responseText` and `ok` are part of the assertion, so I declared them together at the top, also so that it's obvious what are the input parameters for the unit test.

Reply:
`ok` is being used on the `response` object. Cannot be below of it.